docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-services.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Expose multiple services on the same host
+---
+
+This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
+
+The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create those workloads, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
```

Comment:
This is a bit unclear to me.
Does it mean I **must** perform the "Create a workload" tutorial before I can start this tutorial? Or is that optional?

What are the hard prerequisites to perform the following steps? Something like 

> - You have a sample HttpBin service deployment {where?}. 
> - You have a sample Function {where?}.
> - If you don't want to use Kyma's default gateway (kyma-system/kyma-gateway), you know your custom gateway {path?}.

Does the "Create a workload" tutorial show me how to meet those prerequisites?

Reply:
But you do need a workload (no matter if you already have it or create it with the tutorial). And a HttpBin service deployment. And a sample Function too?

Reply:
We used here HttpBin and the sample Function only for demonstration, as they've been used in all the other API exposure tutorials

Reply:
But then what the hard prerequisites...? 😮 
Ah, never mind. I see this sentence appears in all other API Exposure tutorials as well. Let's keep it as is :D